### PR TITLE
Updated the react and react dom version and applied GDS frontend styles

### PR DESCRIPTION
**Pull request by AkhilaHMRC**
#Updated the React & React DOM versions
#Resolved Browser console errors
#Changed the wrappers from MUI to GDS for the CHB app rendering

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,34 +1,76 @@
...
-    </noscript>
+  <body class="govuk-template__body ">
+    <script>
+      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+    </script>
 
-    <div class="flexbox">
-      <!-- React Router will put content in "outlet" -->
-      <div id="outlet" style="width: 100%;"></div>
+    <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
```
Comment (peter-mal) [inline src/index.html:28]: We may want to split out header/footer and other display items to separate react components and reference them in the ChildBen application component.

But I think this is ok for now

**Review by peter-mal — APPROVED** [review]
We may need to revisit some elements of the layout as other work is completed but for now this looks good.

File: src/index.html (hunk 1 above)
Comment (AkhilaHMRC) [reply src/index.html:28]: These are temporary static header and footer for MVP1, these would be removed in the final product, our app would be wrapped / embedded within HMRC header & footer

**Review by peter-mal — APPROVED** [review]
Looks good

**Pull request merged**